src/schema/v2/me/index.ts

```diff
@@ -574,20 +579,23 @@ const MeField: GraphQLFieldConfig<void, ResolverContext> = {
       "identityVerification",
       "unreadNotificationsCount",
       "unseenNotificationsCount",
-      "similarToRecentlyViewedConnection",
       "showsByFollowedArtists",
       "newWorksFromGalleriesYouFollowConnection",
       "myCollectionInfo",
       "watchedLotConnection",
       "userInterestsConnection",
-      "recentlyViewedArtworksConnection",
       "myCollectionConnection",
       "artworkInquiriesConnection",
       "collectionsConnection",
       "auctionResultsByFollowedArtists",
       "myCollectionAuctionResults",
       "newWorksByInterestingArtists",
     ]
+
+    if (xImpersonateUserID) {
```

Comment:
But, for the recently/similar fields that we were short-circuiting above, we _do_ want to short-circuit when impersonation is happening, instead of hitting `meLoader` we'll fetch the id's directly in those resolvers.